Form/JQuery/Type/Select2Type.php

```diff
@@ -89,13 +90,16 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getParent()
     {
-        return $this->widget;
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\\' . ucfirst($this->widget) . 'Type')) {
+            return 'Symfony\Component\Form\Extension\Core\Type\\' . ucfirst($this->widget) . 'Type';
+        }
+        return 'Symfony\Bridge\Doctrine\Form\Type\\' . ucfirst($this->widget) . 'Type';
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
Nope, `getName()` only exists until symfony 2.7 and we only support >=2.8.

But actually I don't really see how this type can work in Symfony3 at the moment. As it has to be referenced by FQCN now how would you distinguish `choice` from `hidden`? @davidzzz did you manage to make it work or are you just looking into this?